src/test/shell/bazel/new_local_repo_test.sh

```diff
@@ -62,6 +62,24 @@ if "$is_windows"; then
   export MSYS2_ARG_CONV_EXCL="*"
 fi
 
+function test_build_file_label_repo_mapping() {
+  mkdir subdir
+  cat > WORKSPACE <<'eof'
+workspace(name='myws')
+load('@bazel_tools//tools/build_defs/repo:http.bzl', 'http_archive')
```

Comment:
nit: why do you need `http_archive()` here?

Reply:
added a comment